### ISSUE TITLE
Wrong data format for automatic detection. `.ai`instead of `.pdf`

### ISSUE THREAD

**Issue by brunto** [y9v/carrierwave-base64#77]
When I submit a PDF file base64 encoded like this `data:application/pdf;base64,JVBERi0xLjQKJfjl0MTGCjEgMCBvYmoKPDwgL...........` the automatic detection change the extension from `.pdf` to `.ai`.

I use:
```
gem 'carrierwave', '~> 1.3.1'
gem 'carrierwave-base64', '~> 2.8.0'
```

Here is my code:

```
    file = params[:attachment][:file]
    @attachment = Attachment.new(attachment_params)
    file = file.gsub('data:application/octet-stream;base64', "data:#{@attachment.mime_type};base64")
    @attachment.attached_file = file
    @attachment.save
```

ps: When I submit an image base64 everything works fine.

Any idea?

**Comment by brunto** [comment]
I found this:

```
MIME::Types['data:application/pdf'].last.preferred_extension
=> "ai"
```

**Comment by brunto** [comment]
See:

**Comment by halostatue** [comment]
This is not a bug in `mime-types`, but in `mime-types-data`. Update that gem to `3.2019.1009` and the issue will fix itself. Resolved in mime-types/mime-types-data#24.
